platform/fabric/core/generic/committer/committer.go

```diff
@@ -308,7 +306,10 @@ func (c *committer) listenTo(txid string, timeout time.Duration) error {
 	for i := 0; i < iterations; i++ {
 		timeout := time.NewTimer(c.pollingTimeout)
 
+		stop := false
 		select {
+		case <-ctx.Done():
+			stop = true
```

Comment:
should there also be a `timeout.Stop()` here?

Reply:
good catch